### PR TITLE
Improve docs for Telegram agent

### PR DESCRIPTION
**Pull request by andrey-yantsen**
Actually users can send messages not only as private messages, but also in groups and channels.
I'm extremely not sure on grammar, sorry :(

### CODE REVIEW

File: app/models/agents/telegram_agent.rb [hunk 1]
```diff
@@ -19,8 +19,15 @@ class TelegramAgent < Agent
...
       1. Obtain an `auth_token` by [creating a new bot](https://telegram.me/botfather).
-      2. Send a private message to your bot by visiting https://telegram.me/YourHuginnBot
-      3. Obtain your private `chat_id` from the recently started conversation by visiting https://api.telegram.org/bot`<auth_token>`/getUpdates
+      2a. If you would like to send messages to a public channel:
+        1. Add your bot to the channel as an administrator
+        2. Set `chat_id` to the name of your channel - e.g. `@YourHugginChannel`
+      2b. If you would like to send messages to a group:
+        1. Add the bot to the group
+        2. Obtain your group `chat_id` from the recently started conversation by visiting https://api.telegram.org/bot`<auth_token>`/getUpdates
+      2c. If you would like to send privetely to you:
```
Comment (cantino) [inline app/models/agents/telegram_agent.rb]: Maybe "If you would like to send messages privately to yourself:"

**Comment by cantino** [comment]
Thanks :)

**Comment by andrey-yantsen** [comment]
Updated

**Comment by cantino** [comment]
Thanks!

**Pull request merged**